vmdb/db/migrate/20150403133609_create_openstack_host_services.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOpenstackHostServices < ActiveRecord::Migration
+  def up
+    create_table :openstack_host_services do |t|
```

Comment:
@Fryguy I was thinking about it this morning, maybe we could make it more generic. I think something like HostServiceGroup (seems like they use ServiceGroup in internal Openstack for similar thing)

(host_services relation is already used on host, it points to system_services, it was renamed for some reason)

And on that STI HostServiceGroupOpenstack, that would hold all OpenStack related methods and queries

At some point in the future we could model ServiceGroup, that would model keystone services and would have relation has_many host_service_groups. (not now)